third_party/rust/bech32/v0_9/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
Unfortunately, we need the exact MIT license that `bech32` is using, including the copyright line here. We can't re-use the same generic license text for that license :(

Same for the other MIT-licensed packages.

Reply:
If we ship what upstream has in their repo, then we are good. :+1: